storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -257,6 +271,10 @@ public String bucketName() {
         return getString(S3_BUCKET_NAME_CONFIG);
     }
 
+    public String storageClass() {
```

Comment:
Let's return `StorageClass` instead, similar to `Region`

Reply:
ack